source/content/guides/decoupled/drupal-backend-starters/04-caching.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Decoupled Drupal Backend Starter for Front-End Sites
+subtitle: Caching Recommendations
+description: Learn about caching recommendations for your Drupal backend starter kit.
+tags: [webops, workflow, decoupled]
+contributors: [backlineint, cobypear, hckia]
+layout: guide
+showtoc: true
+permalink: docs/guides/decoupled/drupal-backend-starters/caching
+anchorid: caching
+contenttype: [guide]
+innav: [true]
+categories: [create]
+cms: [decoupled]
+audience: [development]
+product: [decoupled]
+integration: [--]
+---
+
+Cache settings are configured out-of-the-box for you, but there are circumstances in which you might need to make your own configurations, for example, your CMS is not on Pantheon but your frontend is on Pantheon.
+
+## How it Works
+
+<Partial file="decoupled-caching.md" />
```

Comment:
In my opinion the 'How it works' section jumps right to cache purging, without an overview of the benefits of edge caching in general. I wonder if something could be repurposed from the introductory paragraphs here to cover this:

https://decoupledkit.pantheon.io/docs/backend-starters/decoupled-drupal/caching-considerations
https://decoupledkit.pantheon.io/docs/backend-starters/decoupled-wordpress/caching-considerations